src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/batch_preparers.py

```diff
@@ -61,8 +67,7 @@ def get_account_endpoint(self, *args):
             'accountEndpoint']
         return 'https://' + endpoint
 
-    def batch_cmd(self, cmd, account_info, *args, **kwargs):
-        cmd = cmd.format(*args)
-        cmd = '{} --account-name {} --account-key "{}" --account-endpoint {}'.format(cmd,
-                                                                                     *account_info)
+    def batch_cmd(self, cmd, **kwargs):
+        #cmd = cmd.format(*args)
```

Comment:
One of the static checkers would complain about this.  Do you need it?

Reply:
Nope - just forgot to remove it :)